django_s3/storage.py

```diff
@@ -354,18 +362,20 @@ def isFile(self, path):
             return False
 
     def create_bucket(self, bucket_name):
-        try:
-            self.connection.create_bucket(Bucket=bucket_name)
-        except Exception:
-            pass
-            # logger.exception(f"Failed to create bucket {bucket_name}")
+        if not self.bucket_exists(bucket_name):
```

Comment:
Should also set the default lifecycle policy now that the minio client is installed within the image